numpy/random/mtrand.pyi

```diff
@@ -0,0 +1,132 @@
+import builtins
+from typing import overload, Optional, Sequence, Tuple, TypeVar, Union
+
+from numpy import ndarray
+
+_Size = Union[int, Tuple[int, ...]]
+_T = TypeVar('_T')
+
+class RandomState:
+    def __init__(self, state: Optional[Union[int, ndarray]] = ...) -> None: ...
+    def bytes(self, length: int) -> builtins.bytes: ...
+    def dirichlet(self, alpha: Sequence[float], size: Optional[_Size] = ...
+                  ) -> ndarray: ...
+
+    @overload
+    def choice(self, a: int, *,
+               replace: bool = ..., p: Optional[Sequence[float]] = ...
+               ) -> int: ...
+    @overload
+    def choice(self, a: Sequence[_T], *,
+               replace: bool = ..., p: Optional[Sequence[float]] = ...
+               ) -> _T: ...
+    @overload
+    def choice(self, a: Union[int, Sequence[_T]], size: _Size,
+               replace: bool = ..., p: Optional[Sequence[float]] = ...
+               ) -> ndarray: ...
+
+    @overload
+    def beta(self, a: float, b: float) -> float: ...
+    @overload
+    def beta(self, a: float, b: float, size: _Size) -> float: ...
+    @overload
+    def beta(self, a: ndarray, b: float) -> ndarray: ...
```

Comment:
Unfortunately this isn't quite right -- if you pass in a zero-dimensional ndarray, you get a float back, not a ndarray:
```
In [17]: rs.beta(np.array(2), 2)
Out[17]: 0.3340049666412606
```

This sort of behavior (zero dimensional arrays -> scalars) is unfortunately quite common with NumPy.